chef/cookbooks/pacemaker/attributes/default.rb

```diff
@@ -15,15 +15,19 @@
 
 case node.platform
 when 'suse'
-  default[:pacemaker][:platform][:packages] = %w(pacemaker crmsh)
+  # Install xorg-x11-xauth by default so that 'crm configure graph'
+  # (which uses dotty) will show a nice configuration graph over ssh
+  # with X11 forwarding.
+  default[:pacemaker][:platform][:packages] = %w(
+    pacemaker crmsh xorg-x11-xauth
```

Comment:
Hmm, good point.  Are we also dropping support for SLES11 clusters? Or just adding support for SLES12 clusters?